### ISSUE TITLE
Bitrate/Sample Rate Differences

### ISSUE THREAD

**Issue by TensorTom** [HarryVolek/PyTorch_Speaker_Verification#39]
How much of an impact does the sample rate and bitrate have? My use case is that I need to perform speaker verification across multiple sample rates/codecs such as AMR and EVRC. I will of course be converting them to wav for training but since I will be performing the verification across multiple sample rates, I'm wondering if I should adapt my training set to have varying sample rates as well. Thoughts?

**Comment by HarryVolek** [comment]
It's logical that lowering the sampling rate and bitrate will hurt accuracy, but how much I'm not sure off off the top of my head. I would just try it.

I think it's a good idea to adopt the training set to have multiple sampling rates.
